packages/chevrotain/src/parse/grammar/interpreter.ts

```diff
@@ -422,7 +418,7 @@ export function nextPossibleTokensAfter(
       if (currIdx < tokenVectorLength - 1) {
         const nextIdx = currIdx + 1
         const actualToken = tokenVector[nextIdx]
-        if (tokMatcher(actualToken, prod.terminalType)) {
+        if (tokMatcher!(actualToken, prod.terminalType)) {
```

Comment:
When would `tokMatcher` ever be undefined?

Reply:
validateEmptyOrAlternative passes undefined as tokenMatcher. At runtime it is never undefined in this branch, but we don't explicitly chec that so we need the cast.

Reply:
seems dangerous, maybe pass something valid there instead of try to prove the branch which will throw
and exception is never reached?

Reply:
i fixed that in checks.ts